rollup.config.mjs

```diff
@@ -48,53 +42,97 @@ const node_build = {
       exports: 'named',
     },
   ],
+  external: ['debug'],
   plugins: [
     // Locate modules using the Node resolution algorithm, for using third party modules in node_modules
     nodeResolve({
-      browser: true, // Only resolve browser-compatible modules
+      browser: false, // "browser" properties in package files are ignored
```

Comment:
if it's the default you can remove the prop

Reply:
I was keeping it to be sure the default never changes + it's easier to understand, but I can remove it for simplicity :shrug: